### PR TITLE
Update search docs

### PR DESCRIPTION
**Pull request by kkowalsky**
to be pushed concurrently with https://github.com/pelias/pelias-doc/pull/146

- changes hierarchy of Search TOC

- adds redirect for get started page

- removes get started page because of redundancy

### CODE REVIEW

**Comment by kkowalsky** [comment]
@dianashk are you okay with this? Let me know your thoughts

**Comment by rmglennon** [comment]
preview here: https://precog.mapzen.com/mapzen/mapzen-docs-generator/update-search-docs/documentation/search/

Changes the table of contents to be simpler and removes the "pelias beta" topic from the formatted help -- it's been a year now, so we think this is OK. The source file can still live in pelias-doc for reference.

File: config/search.yml [hunk 1]
```diff
@@ -4,23 +4,24 @@ site_dir: dist-search
...
   - Home: index.md
-  - 'Get started': 'get-started.md'
-  - 'Add search to a map': 'add-search-to-a-map.md'
-  - 'API keys and rate limits': 'api-keys-rate-limits.md'
+  - 'Glossary': 'glossary.md'
   - 'Search': 'search.md'
   - 'Reverse geocoding': 'reverse.md'
   - 'Autocomplete': 'autocomplete.md'
   - 'Place': 'place.md'
+  - 'Tutorial': 'add-search-to-a-map.md'
```
Comment (dianashk) [inline config/search.yml]: `Tutorial` and `Load data from the browser` feel out of place in the order they're in now. Maybe they should go after `HTTP status codes`?

**Comment by dianashk** [comment]
Excellent! Many thanks! 🎉  👍

**Pull request merged**